lib/Doctrine/DBAL/Platforms/PostgreSqlPlatform.php

```diff
@@ -896,7 +886,7 @@ static function ($boolean) : ?int {
      */
     public function convertFromBoolean($item) : ?bool
     {
-        if (in_array($item, $this->booleanLiterals['false'], true)) {
+        if (($this->booleanLiterals[$item] ?? null) === false) {
```

Comment:
Could it be just an `isset()` check?

Reply:
We should read value, not only check that it exists.

Reply:
Yes. An `isset()` check and then read the value. Accessing an element of an array of 12 elements twice is hardly different from doing it once from the performance perspective but makes the logic clear which is more important in this case.